pom.xml

```diff
@@ -123,61 +108,11 @@
             </dependencies>
         </profile>
         <profile>
-            <id>hv-5.4</id>
```

Comment:
Here is the workflow we need IMHO:
- generate the beans once and for all, compatible with all the BV implementations
- be able to run the benchmarks with the same beans on different versions.

I'm not sure what you did is compliant with this workflow?

So maybe we should stick to the validation-api dependencies as 1.1 provided in bean-generator (so that we are sure we only use 1.1 features AND we don't embark the 1.1 jar in the JMH jar).

I'm also wondering if we should make the bean-generator module a bean-generator-bv-1.1 module so that we could create a bv-2.0 in the future to also benchmark the BV 2.0 features.

And we could imagine having 2 different profiles to bundle either benchmarks in the JMH jar.

So for now, we would have `mvn -Phv-6.0-stable` to run the 1.1 benchmarks with HV 6.0 and in the future having `mvn -Pbv-2.0,hv-6.0-stable` to run our hypothetic BV 2.0 benchmarks on HV 6.0.

WDYT?